sway/tree/layout.c

```diff
@@ -346,3 +347,179 @@ void apply_vert_layout(swayc_t *container,
 		*/
 	}
 }
+
+/**
+ * Get swayc in the direction of newly entered output.
+ */
+static swayc_t *get_swayc_in_output_direction(swayc_t *output,
+		enum movement_direction dir, struct sway_seat *seat) {
+	// XXX is this really a seat function or can we do it with the default
+	// seat?
+	if (!output) {
+		return NULL;
+	}
+
+	swayc_t *ws = sway_seat_get_focus_inactive(seat, output);
+	if (ws->type != C_WORKSPACE) {
+		ws = swayc_parent_by_type(ws, C_WORKSPACE);
+	}
+
+	if (ws && ws->children->length > 0) {
+		switch (dir) {
+		case MOVE_LEFT:
+			// get most right child of new output
+			return ws->children->items[ws->children->length-1];
+		case MOVE_RIGHT:
+			// get most left child of new output
+			return ws->children->items[0];
+		case MOVE_UP:
+		case MOVE_DOWN:
+			{
```

Comment:
This probably comes from old code but can you move this { over to the same line as the case and deindent the whole thing so it's not as crazy